src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1792,45 +1793,31 @@ public static function addEnclosuresToTemplate($objTemplate, $arrItem, $strKey='
 	/**
 	 * Set the static URL constants
 	 *
-	 * @param PageModel $objPage An optional page object
+	 * @deprecated Deprecated in Contao 4.5, to be removed in Contao 5. Use the asset contexts instead.
```

Comment:
Since we have deprecated the method, we should no longer use it. Otherwise, if we still need to use it, we should not deprecate it. 😉